packages/block-library/src/image/block.json

```diff
@@ -83,7 +83,8 @@
 		},
 		"__experimentalBorder": {
 			"radius": true
-		}
+		},
+		"__experimentalSelector": ".wp-block-image img"
```

Comment:
Not sure if this is a good idea?

Reply:
Do the other block supports still work? If we don't to implement this PR we'd need to implement "selectors per property" first, that we talked about at https://github.com/WordPress/gutenberg/issues/28913

Reply:
I think the only one that would be affected would be border radius, which seems to still work fine.

Reply:
This is problematic.

These are the testing steps:

- Using the empty theme, set the border-radius for the image block to 0px in the theme.json 
- Create a post and add an image. Set its border-radius to any value.

I expected the image to change the border but it didn't. The theme style rule has higher specificity than the block's.

The border-radius rules were added in https://github.com/WordPress/gutenberg/pull/27667/files#r699061648 I'm going to pull @aaronrobertshaw thoughts on this. I'd think the fix for this would be to serialize the border-radius directly to the `img` element, using `__experimentalSkipSerialization`. Once we have fixed that in a separate PR, we could continue with this one.

Reply:
Thanks for the ping. 

`__experimentalSkipSerialization` didn't exist yet when border-radius support was originally added to the image block. Now that it does, there's no reason we can't move the application of the border styles to the inner image element.

In fact, there is already an [open PR](https://github.com/WordPress/gutenberg/pull/31366) that does this as part of adopting border color and width for the image block.